.github/workflows/trigger_prerelease.yml

```diff
@@ -14,6 +14,8 @@ jobs:
     uses: newrelic/coreint-automation/.github/workflows/trigger_prerelease.yaml@v1
     with:
       rt-included-files: go.mod,go.sum,build/Dockerfile
+      bot_email: '${{ vars.K8S_AGENTS_BOT_EMAIL }}'
+      bot_name: '${{ vars.K8S_AGENTS_BOT_NAME }}'
```

Comment:
Are these suppose to be environment variables or secrets?

Reply:
Variables, matching what is defined in [coreint-automation](https://github.com/newrelic/coreint-automation/blob/main/.github/workflows/trigger_prerelease.yaml#L15-L30)